pkgs/top-level/python-packages.nix

```diff
@@ -6880,6 +6920,34 @@ in {
 
   jinja2_pluralize = callPackage ../development/python-modules/jinja2_pluralize { };
 
+  jira = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.